pkgs/development/python-modules/flask-security-too/default.nix

```
@@ -59,6 +59,15 @@ buildPythonPackage rec {
     hash = "sha256-we2TquU28qP/ir4eE67J0Nlft/8IL8w7Ny3ypSE5cNk=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fixes-for-py_webauthn-2.0.patch";
+      url = "https://github.com/Flask-Middleware/flask-security/commit/2abd4e232b7943628f89d5d8033d64896286230f.patch";
```

```suggestion
      url = "https://github.com/Flask-Middleware/flask-security/commit/5725f7021343567ec0b25c890e859f4e84c93ba6.patch";
```

that is the merged hash which has a reference to the PR on https://github.com/Flask-Middleware/flask-security/commit/5725f7021343567ec0b25c890e859f4e84c93ba6